js/buda.js

```diff
@@ -389,8 +389,8 @@ module.exports = class buda extends Exchange {
         const marketId = this.safeString (ticker, 'market_id');
         const symbol = this.safeSymbol (marketId, market, '-');
         const lastPrice = this.safeValue (ticker, 'last_price', []);
-        const last = this.safeNumber (lastPrice, 0);
-        const percentage = this.safeNumber (ticker, 'price_variation_24h');
+        const last = this.safeString (lastPrice, 0);
+        const percentage = this.safeString (ticker, 'price_variation_24h');
```

Comment:
We lost percentage multiplication by a hundred here, and we need to do it with the `Precise` class in string-based mode.